src/components/CharacterLimitMainText/CharacterLimitMainText.tsx

```diff
@@ -0,0 +1,18 @@
+import CharacterComp from './components/CharacterComp/CharacterComp';
+
+import scss from './CharacterLimit.module.scss';
+
+import { ICharacterLimitMainTextProps } from './CharacterLimitMainText.types';
+
+function CharacterLimitMainText(props: ICharacterLimitMainTextProps) {
+  return (
+    <div className={scss.compost}>
+      <CharacterComp
+        value={props.value}
+        socialList={props.socialList}
+      ></CharacterComp>
```

Comment:
Se nao tem necessidade de children, pode ser self closed.
```suggestion
      <CharacterComp
        value={props.value}
        socialList={props.socialList}
      />
```

Reply:
workado